src/scripts/merge_tools/gitmerge_ort.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env sh
 
-# usage: ./gitmerge-ort.sh <clone_dir> <branch-1> <branch-2>
+# usage: ./gitmerge_ort.sh <clone_dir> <branch-1> <branch-2>
 
 MERGE_SCRIPTS_DIR="$(cd "$(dirname "$0")" && pwd -P)"
 clone_dir=$1
 branch1=$2
 branch2=$3
 strategy="-s ort"
-"$MERGE_SCRIPTS_DIR"/gitmerge.sh "$clone_dir" "$branch1" "$branch2" "$strategy"
+if ! "$MERGE_SCRIPTS_DIR"/gitmerge.sh "$clone_dir" "$branch1" "$branch2" "$strategy"; then
```

Comment:
@benedikt-schesch Why is this necessary?  The script should exit with the same status as the last command in it, so this `if` construct seems unnecessary (throughout).